packages/react/src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { Breadcrumbs } from '.';
+
+export default {
+  title: 'Komponenter/Breadcrumbs',
+  component: Breadcrumbs.Root,
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg'],
+      control: { type: 'radio' },
+    },
+  },
+  args: {
+    size: 'md',
+  },
```

Comment:
We don't use argTypes as it leads to unnecessary maintenance and prone for manual errors. We'll fix Storybook in a separate PR for it to automatically detect the props table.

Reply:
Right, so we should remove it in Chip, Group, Removable and Toggles too? ☺️

Reply:
Jepp!